spec/Spec/BlsHelpers.cry

```diff
@@ -33,6 +40,12 @@ bls_modular_inverse x = toIntegral (modular_inverse x') where
 /**
  * Divide two field elements: x by y
  * @see https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/polynomial-commitments.md#div
+ * ```repl
+ * :prove bls_div 1 2 == 0x39f6d3a994cebea4199cec0404d0ec02a9ded2017fff2dff7fffffff80000001
```

Comment:
Reason why these are `:prove`?

Reply:
Looks like I missed these. Thanks for catching it. I'll add a commit.